packaging/docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM centos:7
+LABEL maintainer="tdockendorf@osc.edu; johrstrom@osc.edu"
+
+# setup the ondemand repositories
+COPY files/ondemand-web.repo /etc/yum.repos.d/ondemand-web.repo
+COPY files/ondemand-ci-web.repo /etc/yum.repos.d/ondemand-ci-web.repo
+RUN curl https://yum.osc.edu/ondemand/RPM-GPG-KEY-ondemand -o /etc/pki/rpm-gpg/RPM-GPG-KEY-ondemand
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-ondemand
+
+# update
+RUN yum install -y centos-release-scl 
+RUN yum -y update
```

Comment:
Every RUN will create a layer, so the YUM cache from each RUN will create a large layer. If your going to execute yum commands I'd recommend concatenating them all into a single RUN command like we do in ood-images. This will reduce the final image size as long as the last command in RUN is "yum clean all".

Reply:
These are all in one RUN command now.